### PR TITLE
Add doctors retrieving

### PR DESCRIPTION
**Pull request by m-grzesiak**

### CODE REVIEW

File: src/main/java/pl/michal/grzesiak/luxmed/parameters/Doctors.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package pl.michal.grzesiak.luxmed.parameters;
+
+import lombok.Value;
+
+import java.util.Set;
+
+@Value
+public class Doctors {
+
```
Comment (mwojtaczka) [inline src/main/java/pl/michal/grzesiak/luxmed/parameters/Doctors.java]: Why is it Doctors when it contains both, doctors and payers?

File: src/test/java/pl/michal/grzesiak/luxmed/LuxmedTest.java [hunk 2]
```diff
@@ -108,4 +111,26 @@ void shouldReturnServicesAndClinicsInCity() {
...
+        long cityId = 1;
+        long serviceId = 1;
+        long clinicId = 1;
+
+        Doctors doctors = luxmed.reservationCriteria()
+                .token(ACCESS_TOKEN)
+                .getDoctors(cityId, serviceId, clinicId);
+
+        assertAll(
+                () -> assertThat(doctors).isNotNull(),
```
Comment (mwojtaczka) [inline src/test/java/pl/michal/grzesiak/luxmed/LuxmedTest.java]: It is better to keep that assertion out of the assertAll block, when that one fails all others will fail as well. The response from the test will contain unnecessary comments.

File: src/main/java/pl/michal/grzesiak/luxmed/parameters/Doctors.java (hunk 1 above)
Comment (m-grzesiak) [reply src/main/java/pl/michal/grzesiak/luxmed/parameters/Doctors.java]: will be fixed later on

File: src/test/java/pl/michal/grzesiak/luxmed/LuxmedTest.java (hunk 2 above)
Comment (m-grzesiak) [reply src/test/java/pl/michal/grzesiak/luxmed/LuxmedTest.java]: will be fixed later on

**Pull request merged**